src/publicodes-state/helpers/migrateSimulation.ts

```diff
@@ -33,5 +33,17 @@ export function migrateSimulation(
     delete simulation.poll
   }
 
+  // If the computedResults object does not take multiple metrics into account, we add them
+  if ((simulation.computedResults as any)?.bilan !== undefined) {
+    const newComputedResults = {
+      carbone: JSON.parse(JSON.stringify(simulation.computedResults)),
```

Comment:
The serialization / deserialization is needed ??

Reply:
Non effectivement. Je ne sais pas pourquoi c'est la